govcd/api.go

```diff
@@ -656,8 +662,6 @@ func executeRequestCustomErr(pathURL string, params map[string]string, requestTy
 		req.Header.Add("Content-Type", contentType)
 	}
 
-	setHttpUserAgent(client.UserAgent, req)
```

Comment:
This was extraneus setting, because `newRequest` function already added that header.